packages/payment-integration-test-framework/src/fixture/pageObject/Checkout.ts

```diff
@@ -32,8 +34,10 @@ export class Checkout {
     }
 
     async close(): Promise<void> {
```

Comment:
Do we need to have a way to reset all the variables to `false`?

Reply:
`!this.harFileName || !this.usedPreset || !this.openedCheckout` serves as a reminder for a developer to set a proper checkout envirnment and record a test. They should be always`true` at the end of every test.

Since I don't expect a complicated usa case, such as changing payment configuration in the middle of a test, it should be sufficient to check the essential functions have run at least once for now.